src/fi/bitrite/android/ws/activity/Maps2Activity.java

```diff
@@ -708,7 +708,7 @@ public void showMultihostSelectDialog(final ArrayList<HostBriefInfo> hosts) {
         alertDialogBuilder.setCustomTitle(customTitleView);
 
         alertDialogBuilder
-                .setNeutralButton(R.string.ok, new DialogInterface.OnClickListener() {
+                .setNegativeButton(R.string.ok, new DialogInterface.OnClickListener() {
```

Comment:
Moves the button from the bottom left to bottom right. Also, imo, your are 'cancelling' your action if you click this, more then just 'ok'.